docs/wiki/framework/wardrobe-framework.md

```diff
@@ -0,0 +1,284 @@
+---
+layout: wiki
+title: Wardrobe Framework
+description: Explains the wardrobe system and how to make a compat.
+group: framework
+order: 5
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 20
+  patch: 0
+---
+
+
+## 1. Overview
+
+The Wardrobe Addon gives the player the opportunity to change/modify their current wearables into different variants.
+
+For example, if a uniform has a normal variant and a "Rolled-Up Sleeves" variant, the player will be able to use ether an ACE Self Interaction or the CBA Context Menu in the Inventory.
+
+Since there is no reliable, common patterns in terms of class inheritance, not even within the same DLC, each possible variant has to be defined individually within `configFile >> "ace_wardrobe"`.
+
+### 1.1 Components
+
+Components are an optional assistance to define a group of variants where a variant is made of one or more components.
+For example, a bandanna with sunglasses would have the the bandanna and the sunglasses as components, while the bandanna and sunglasses themself only have themselves as a component.
+
+If the player switches from a variant with more components to a variant with less components, the surplus components will be deposited in their inventory. Vise Versa, if the player wants to change from a variant with less components to a variant with more components, they are required to have the missing components in their inventory, which will be removed on conversion.
+
+More examples can be found below.
+
+### 1.2 Config Guidelines
+
+- A Container (Uniform, Vest, Backpack) should never change its `maxLoad`, unless there is a really good reason for it. This will result in the loss of items.
+- Variants should not "magically" add/remove parts to/from themselves, unless it's handled through a component, especially when they add protection or functionality.
+  - Do: Uniforms with and without Gloves should be fine in most cases, as they are cosmetical and don't break continuity/logic.
+  - Do: Helmets with integral flip-down visors.
+  - Don't: Helmet with additional armor or face protector without the requirement of some component that will be removed from the inventory.
+- There should not be a difference in weight or container capacity between the different variants unless components get added/removed.
+  - For Example: `Mass of Helmet with Goggles = Mass of Helmet + Mass of Goggles.`
+
+## 2. Config
+
+Only directly defined Subclasses will be taken into account. A fully inherited subclass will be ignored and will not provide any interaction. The `ace_wardrobe` properties are to be found as a subclass of the item itself. Multiple base configs are designed at root of `configFile`.
```

Comment:
I have noticed a lot of capitalised words that shouldn't be. I understand in some instances, but imo it's not warranted in most cases I've seen so far.
```suggestion
Only directly defined subclasses will be taken into account. A fully inherited subclass will be ignored and will not provide any interaction. The `ace_wardrobe` properties are to be found as a subclass of the item itself. Multiple base configs are designed at root of `configFile`.
```